website/content/partials/telemetry-metrics/vault/core/step_down.mdx

```diff
@@ -0,0 +1,9 @@
+### vault.core.step_down ((#vault-core-step_down))
+
+Metric type | Value | Description
+----------- | ----- | -----------
+summary     | ms    | Time required to step down cluster leadership
+
+Step down time is an important health metric for your Vault installation. We
```

Comment:
Fair. I included it because the metric had a note about it being important in the current doc. But if that's not an appropriate recommendation, I'll remove the callout.